transcendence-app/src/app.module.ts

```diff
@@ -1,10 +1,20 @@
 import { Module } from '@nestjs/common';
-import { AppController } from './app.controller';
-import { AppService } from './app.service';
+import { ConfigModule } from '@nestjs/config';
+import * as Joi from 'joi';
+import { AuthModule } from './auth/auth.module';
 
 @Module({
-  imports: [],
-  controllers: [AppController],
-  providers: [AppService],
+  imports: [
+    ConfigModule.forRoot({
+      isGlobal: true,
+      cache: true,
+      validationSchema: Joi.object({
```

Comment:
I have read [here](https://dev.to/yawaramin/comment/dfhg) that it's okay to mix and match. Typescript provides compile time validation, joi provides runtime validation.

But mostly I think we should let ourselves a bit of freedom to experiment and add random modules just for the sake of discovering them, even if they pollutes our code a bit.

If it enables you, I think it's okay to leave it there for now. If we don't like it, we can remove it later on.

That's my opinion.

Reply:
Done, I've replaced this package with the `class-validator` since we are also using it in other places.

As @Damrod says, it's nice to have compile-time and runtime validation.